### DISCUSSION TITLE
Build Zeal for Mac OS X — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#252] [Announcements]
https://mazhuang.org/2016/01/16/build-zeal-for-mac-osx/

**Comment by liejuntao001** [comment]
Thanks, it helped a lot. You version can no longer download docsets. 
I compiled version 0.5.0 and shared here: 
链接: https://pan.baidu.com/s/1gJXO7H_TFNGmioYUpIj0tQ 密码: hpwx
After 0.5.0 the author removed qmake support.
